### DISCUSSION TITLE
Add option radius on Bottom drawer

### DISCUSSION THREAD

**Discussion by Gabrielfusf** [mantinedev/mantine#5001] [Feature requests]
### What package has an issue

@mantine/core

### Describe the bug

Not is bug. 
Feature add radius on drawer bottom

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

7

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
You can override Drawer and other component styles with Styles API.
